src/ORM/Locator/TableLocator.php

```diff
@@ -307,4 +344,18 @@ public function remove($alias)
             $this->_fallbacked[$alias]
         );
     }
+
+    /**
+     * Adds a namespace where tables should be looked for.
```

Comment:
Having a usage e.g. would be nice here to clarify that one needs to specify namespace without the top level. For e.g. just `Infrastructure\Table` namespace `TestPlugin\Infrastructure\Table`.

Reply:
Yes. Maybe "namespace" is a wrong word as it suggests the whole namespace should be used. Maybe "location" is better?

Reply:
Yes, "location" or "path" would be better.

Reply:
Since it's in TableLocator I guess `$location` is nicer.